src/witan/app/forecast.clj

```diff
@@ -311,14 +322,14 @@
    (go ;; temporary solution
      (log/info "Starting to run model: " (:model_id forecast) (:name model) (str "v" (:version model)))
      (try
-       (let [outputs  (mex/execute-model forecast model)]
+       (let [outputs (mex/execute-model forecast model)]
          (if-let [error (:error outputs)]
            (process-error! forecast error)
-           (let [data      (into {} (->> (first outputs)
-                                         (map #(process-output-data! % (:public? forecast)))))]
-             (log/info "Finished running model " (:model_id forecast) "-" (count data) "output(s) returned." outputs)
+           (let [_ (log/info "Finished running model" (:model_id forecast) "- processing...")
+                 data (into {} (map #(process-output-data! % (:public? forecast))) outputs)]
+             (log/info "Finished processing model " (:model_id forecast) "-" (count data) "output(s) returned.")
              (conclude-forecast! (assoc (->Forecast forecast) :outputs data)))))
-       (catch Exception e (log/info "Error around model " (:model_id forecast) " " (.getMessage e) (clojure.stacktrace/print-stack-trace e)))))))
```

Comment:
I purposely changed this to "error around model" because there is already a catch in the model execution itself, which also gets logged.  This catch should only catch errors 'around' the model, if that makes sense.